sql/core/src/test/scala/org/apache/spark/sql/execution/metric/SQLMetricsSuite.scala

```diff
@@ -552,11 +552,13 @@ class SQLMetricsSuite extends SparkFunSuite with SQLMetricsTestUtils with Shared
       // The execution plan only has 1 FileScan node.
       val df = spark.sql(
         "SELECT * FROM testDataForScan WHERE p = 1")
-      testSparkPlanMetrics(df, 1, Map(
```

Comment:
shall we update `testSparkPlanMetrics` to check driverMetrics as well?

Reply:
Yep, actually the changing [here](https://github.com/apache/spark/pull/23396/files#diff-dc5c7a41fbb7479cef48b67eb41ad254R56) has made `testSparkPlanMetrics` able to check drvierMetrisc, I'll revert the changes here. Done in 10df5b1.